float64[] obstacle
float64[] vertices
float64[] volumes
float64[] posx
float64[] servoing_point
float64[] robot
